dev/packages/alpha/cisco/0.0.2/dataset/asa/agent/stream/log.yml.hbs

```diff
@@ -0,0 +1,9 @@
+input: log
```

Comment:
@ph Should this be `logs`

Reply:
@ruflin are you sure about this? I've seen `input: log` in other places: https://github.com/elastic/package-registry/blob/master/dev/packages/example/nginx/0.0.2/dataset/access/agent/stream/stream.yml.hbs

Reply:
Hm, we do some matching here: https://github.com/elastic/package-registry/blob/master/util/package.go#L355 I would assume it should not work. I need to dig into this.

We probably should also validate this to make sure a stream can only use defined inputs.

Reply:
I think this form is applied in integrations and it works, so you find that there is an issue around, feel free to create a general issue.

Reply:
I was digging into this and realised, I think we don't need it at all here. By now we have in the manifest a path which points to the stream template. So we know which input it belongs to.

Could you try to remove it for example from nginx and see if things still work as expected?